src/Generator/Entity/ModelEntity.cs

```diff
@@ -10,18 +10,17 @@ internal class ModelEntity : ClassEntity
     internal ModelEntity(DTInterfaceInfo interfaceInfo, ModelGeneratorOptions options) : base(options)
     {
         ModelId = interfaceInfo.Id;
-        Properties = interfaceInfo.Contents.Values
-                            .Where(c => c.Id.Labels.Contains(Name) && c.EntityKind == DTEntityKind.Property && c is DTPropertyInfo)
-                            .Select(c => (DTPropertyInfo)c);
         Name = GetClassName(interfaceInfo.Id);
         Parent = interfaceInfo.Extends.Count() > 0 ? GetClassName(interfaceInfo.Extends.First().Id) : nameof(BasicDigitalTwin);
         FileDirectory = ExtractDirectory(ModelId);
-        var properties = GetContentInfo<DTPropertyInfo>(interfaceInfo);
-        PropertyContent.AddRange(properties.Select(CreateProperty));
-        var relationships = GetContentInfo<DTRelationshipInfo>(interfaceInfo);
-        PropertyContent.AddRange(relationships.Select(CreateProperty));
+        var contents = interfaceInfo.Contents.Select(c => c.Value).Where(c => c.Id.Labels.Contains(Name) && c is not DTCommandInfo);
```

Comment:
Can you re-check the diff here? Unsure why these changes happened...

The `if (commands. Count() > 0)` wrapping `CommandContent.AddRange(...)` shouldn't be necessary either.

Reply:
Thanks for catching this!! Its redundant. When I was moving things around after Osama's changes, I must've added this change and not removed it.

As for the other change, I moved Property and Relationship under one call to already exisiting code and since it'll be a pretty large change to include Command in that class, I've kept the command piece separate.